ratpack-manual/src/content/chapters/97-java9-support.md

```diff
@@ -0,0 +1,92 @@
+# Java 9 Support
+
+This is a list of known issues, caveats, and workarounds to use early Java 9+ support. While Java 9 support is far from
+complete, core functionality appears to work with only a few warnings and messages here and there.
+
+## Known Library Caveats
+
+The following are known version conflicts or requirements for Ratpack to run seamlessly on Java 9.
+
+* `ratpack-groovy` requires Groovy 2.5.2 or higher per issue [#1411](https://www.github.com/ratpack/ratpack/issues/1411)
+* `ratpack-groovy-test` requires Groovy 2.5.2 or higher per issue [#1411](https://www.github.com/ratpack/ratpack/issues/1411)
+
+## Known Issues
+
+The following are issues caused by Java 9 and the underlying components to Ratpack at this time. Workarounds will be provided in each case
+
+* ratpack-handlebars / apache commons lang3 conflicts  
```

Comment:
This should be able to be removed due to the PR being merged. It's also not a Java 9 issue. This would break on Java 8 as well.

Reply:
Okay, I'll clean this one up! Thanks for the heads up on it